### DISCUSSION TITLE
Digital/lab6/tasks/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#21] [Announcements]
# Digital/lab6/tasks/

https://soc.ustc.edu.cn/Digital/lab6/tasks/

**Comment by Thebunniest** [comment]
必做部分的ALU一题中的“stl”/“stlu”是不是应该写作“slt”/“sltu”？

**Comment by Thebunniest** [comment]
选择性必做3的Shifter模块可能少了个endmodule（噗，我怎么觉得我一直搁这一眼丁真呢

**Reply by wintermelon008** [reply]
挺好的，有些地方我们校对时确实看不出来（

**Comment by woqinmian** [comment]
选择性必做3的仿真文件中src1=32‘h00000错了，应该是src1=5’b00000
